contributors/design-proposals/storage/mount-options.md

```diff
@@ -72,7 +72,8 @@ metadata:
 provisioner: kubernetes.io/glusterfs
 parameters:
   type: gp2
-  mountOptions: "auto_mount"
+mountOptions:
+  - auto_mount
```

Comment:
This makes sense to me but I'll let @gnufied (author of the design proposal) lgtm this.